Fugro/G2O/Fugro.G2O.nuspec

```diff
@@ -14,18 +14,22 @@
       <files include="**" buildAction="None" copyToOutput="true" flatten="true" />
     </contentFiles>
     <dependencies>
-      <group targetFramework=".NETFramework4.5" />
-      <group targetFramework=".NETCoreApp3.1" />
-    </dependencies>
+      <group targetFramework=".NETFramework4.8" />
+		  <group targetFramework=".NETCoreApp3.1" />
+		  <group targetFramework=".NET6.0" />
+	</dependencies>
   </metadata>
   <files>
-    <file src="bin\x64\clr-release\Fugro.G2O.dll" target="lib\net45" />
-    <file src="bin\x64\clr-release\Fugro.G2O.xml" target="lib\net45" />
+    <file src="bin\x64\clr-release\Fugro.G2O.dll" target="lib\net48" />
+    <file src="bin\x64\clr-release\Fugro.G2O.xml" target="lib\net48" />
     <file src="bin\x64\Release\Fugro.G2O.dll" target="lib\netcoreapp3.1" />
     <file src="bin\x64\Release\Fugro.G2O.xml" target="lib\netcoreapp3.1" />
-    <file src="bin\x64\Release\Ijwhost.dll" target="contentFiles\any\netcoreapp3.1" />
+    <file src="bin\x64\Release\Ijwhost.dll" target="lib\netcoreapp3.1" />
+    <file src="bin\x64\net6-release\Fugro.G2O.dll" target="lib\net6.0" />
+    <file src="bin\x64\net6-release\Fugro.G2O.xml" target="lib\net6.0" />
+    <file src="bin\x64\net6-release\Ijwhost.dll" target="lib\net6.0" />

```

Comment:
`Ijw` or `ijw`? May matter on linux. "Dependencies" seemed to indicate the latter.

Reply:
Ijwhost is not "shipped" in this repo. VS2022 just seems to provide one when you build? So I can't rename it? It is with a capital... Maybe this is the reason it doesn't run on linux?

Reply:
I'd say --> another PR for v2.0.3 when linux becomes necessary

Reply:
OK!